src/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -364,6 +364,13 @@ public static unsafe ushort RhGetComponentSize(EETypePtr ptrEEType)
             return pEEType->ComponentSize;
         }
 
+        [RuntimeExport("RhGetBaseSize")]
+        public static unsafe uint RhGetBaseSize(EETypePtr ptrEEType)
```

Comment:
Not directly related to this PR, but I would like to phase out these 'trivial EEType accessor APIs'.  The EEType structure is, essentially, a binary contract and the upper layer runtime code (i.e. type loader layer) can know everything there is to know about EETypes, so there's not really any need, long term, to have these trivial accessors -- they provide no value.